### PR TITLE
Fix inconsistencies in backup state flags

### LINKED ISSUE #1740: Inconsistencies in backup state flags
Follow-up to https://github.com/w3c/webauthn/pull/1695#pullrequestreview-1001739999 :

 1. [§6.1.3. Credential Backup State](https://w3c.github.io/webauthn/#sctn-credential-backup) states:

    >The following is a non-normative, non-exhaustive list of how [Relying Parties](https://w3c.github.io/webauthn/#relying-party) might use these [flags](https://w3c.github.io/webauthn/#flags):

    However, the described list contains normative SHOULDs.

 2. [§6.1.3. Credential Backup State](https://w3c.github.io/webauthn/#sctn-credential-backup) states:
  
    >It is RECOMMENDED that [Relying Parties](https://w3c.github.io/webauthn/#relying-party) store the most recent value of these [flags](https://w3c.github.io/webauthn/#flags) with the [user account](https://w3c.github.io/webauthn/#user-account) for future evaluation.

    Which indicates storing both the `BE` and the `BS` flags. However, [§7.1. Registering a New Credential](https://w3c.github.io/webauthn/#sctn-registering-a-new-credential) instructs to store `BS` but not `BE`.

### PR DESCRIPTION
**Pull request by emlun**
Fixes #1740. This depends on PR #1771.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1772.html" title="Last updated on Sep 21, 2022, 2:22 PM UTC (196f075)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1772/573b1c2...196f075.html" title="Last updated on Sep 21, 2022, 2:22 PM UTC (196f075)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -3848,11 +3847,11 @@ laid out as shown in <a href="#table-authData">Table <span class="table-ref-foll
                     - `1` means the user is [=user verified|verified=].
                     - `0` means the user is not [=user verified|verified=].
                 - Bit 3: [=Backup Eligibility=] (<dfn>BE</dfn>).
-                    - `1` means the [=public key credential source=] is allowed to be backed up.
-                    - `0` means the [=public key credential source=] is not allowed to be backed up.
+                    - `1` means the [=public key credential source=] is is [=backup eligible=].
```
Comment (timcappalli) [inline index.bs:3850]: double "is"

**Review by timcappalli — CHANGES_REQUESTED** [review]
Outside of the extra "is" on line 3850, LGTM!

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs]: Good catch, thanks!

**Review by MasterKale — APPROVED** [review]

**Review by pascoej — APPROVED** [review]
This looks good.

**Review by akshayku — APPROVED** [review]

**Comment by emlun** [comment]
Thanks @timcappalli, but as noted in the top post I'll keep this as a "draft" PR until #1771 is merged.

**Review by timcappalli — APPROVED** [review]
lgtm

**Pull request merged**